go.mod

```diff
@@ -3,20 +3,20 @@ module github.com/tonistiigi/go-actions-cache
 go 1.22
 
 require (
-	github.com/Azure/azure-sdk-for-go/sdk/storage/azblob v1.5.0
+	github.com/Azure/azure-sdk-for-go/sdk/storage/azblob v1.6.0
 	github.com/dimchansky/utfbom v1.1.1
 	github.com/golang-jwt/jwt/v5 v5.2.1
 	github.com/pkg/errors v0.9.1
-	github.com/stretchr/testify v1.9.0
-	golang.org/x/sync v0.8.0
+	github.com/stretchr/testify v1.10.0
+	golang.org/x/sync v0.10.0
```

Comment:
Would be fine for buildkit, this dep is aligned: https://github.com/moby/buildkit/blob/67023657b39cb5abbade0ea0d863809b2b2f0363/go.mod#L100